.ci/ci_job_flags.sh

```diff
@@ -178,6 +178,14 @@ case "${CI_JOB}" in
 	export KUBERNETES="yes"
 	export experimental_kernel="true"
 	;;
+"EXTERNAL_CRIO")
```

Comment:
please, just ensure we have all thje CI flags initalised as expected by calling `init_ci_flags` as the first thing.

Reply:
right - my bad.
Just fixed it.